docs/site/tutorials/a_swift_tour.ipynb

```diff
@@ -1835,26 +1879,37 @@
     },
     {
       "metadata": {
-        "id": "zgtd86KUp5qa",
+        "id": "Vz405lhC35ZX",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "var queen = Rank.two // Oops! Wrong value!\n",
+        "queen = .queen"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "9R71YWaE4L_W",
         "colab_type": "text"
       },
       "cell_type": "markdown",
       "source": [
-        "Use the `init?(rawValue:)` initializer to make an instance of an enumeration from a raw value. It returns either the enumeration case matching the raw value or `nil` if there is no matching `Rank`."
+        "Notice above that in the second line, `Rank.queen` is referred to by the abbreviated form `.queen` because the type of the variable `queen` is already known to be a `Rank`. You can use the abbreviated form anytime the value’s type is already known."
```

Comment:
Similar to the cell below where the dot notation is used on `self` in the switch case statement, I thought I should put a simple example on a variable type.